src/cmds/qstat.c

```diff
@@ -102,6 +102,8 @@ char *cnvt_est_start_time(char *start_time, int shortform);
 #define HPCBP_EXEC_TAG  "jsdl-hpcpa:Executable"
 #define CHAR_LINE_LIMIT 78 /* maximum number of characters which can be printed on a line */
 
+#define DISPLAY_TRUNC_FLAG '*'
```

Comment:
This really isn't a flag.  A flag changes how something works.  This is just a character that indicates something.  Maybe DISPLAY_TRUNC_CHAR?